recipes/js-format

```diff
@@ -0,0 +1,3 @@
+(js-format :fetcher github
+           :repo "futurist/js-format.el"
+           :files ("*.js" "*.el" "*.json"))
```

Comment:
`:defaults` is preferable to `"*.el"`. 

https://github.com/melpa/melpa#recipe-format